### DISCUSSION TITLE
ЛР 6 для ИВТ. Ошибка в схеме модуля CSR

### DISCUSSION THREAD

**Discussion by mrz25** [MPSU/exAPS#53] [Ошибки]
![image](https://user-images.githubusercontent.com/71031775/206928012-7e72a86b-7346-4d07-b4e0-b26405dbd52c.png)
![image](https://user-images.githubusercontent.com/71031775/206928100-9ec818e9-4623-4db2-aa58-af1e7e2cbb3e.png)
у вас ошибка в рисунке, у инструкции csrrc func3 = 011, соответственно на мультиплексоре должен соответствовать значению 3, а не 2 как у вас

**Comment by HepoH3** [comment]
Создал issue, потом исправим, спасибо!
Напиши в телегу по поводу доп балла.
